Section_II/competition_and_trophies.tex

```diff
@@ -274,53 +274,107 @@
 \bigskip
 
 
-{\bfseries Free-kick procedure}
+{\bfseries \removed{Free-kick}\added{Game interruption} procedure}
+
+\added{
+This procedure applies for throw-ins, corners, goal kicks, direct free-kicks,
+indirect free-kicks and penalties called during a game.
+It ensures a minimal distance of $D$ between objects that were moved during
+this phase.
+For KidSize, $D$ is 50 centimeters and for AdultSize, $D$ is 1 meter.
+}
 
 \begin{enumerate}
-\item The referee blows the whistle, announces the offence and the free kick
+\item The referee \removed{blows the whistle,} announces the offence and the
+      \added{game interruption}\removed{free kick}
       (e.g 'Pushing Red - Direct Free-Kick Blue')
-\item The assistant referee who operates the game controller clicks on "Direct / Indirect Free Kick" Blue/Red.
-\item The referee places the ball depending on the call and announces "Free Kick Ready".
+\item \removed{The assistant referee who operates the game controller clicks on
+    "Direct / Indirect Free Kick" Blue/Red.}
+  \added{The referee communicates the game interruption and to which team it is
+    awarded to the game controller.}
```

Comment:
Add: "or the assistant referee operating the game controller in a physical competition.